backend/project/settings/local_template.py

```diff
@@ -15,13 +15,12 @@
     ]
```

Comment:
It is just a template for local settings, you can copy it to make a file for your own local settings to override settings in base.py